modular_skyrat/code/_HELPERS/roundend.dm

```diff
@@ -0,0 +1,27 @@
+/datum/controller/subsystem/ticker/declare_completion()
+	..()
+	for(var/M in GLOB.player_list)
+		var/mob/m = M
+		if (!m.client)
+			continue
+		m.client.verbs += /client/proc/eorg_teleport
```

Comment:
You might be better off making this an action instead of a given proc so players don't have to click around in tabs.

Reply:
If the players have the preference enabled, they're moved automatically without having to do anything, they don't have to use the verb in the OOC tab, that verb is only for:
1. Some accident where the teleport fails to work as a back-up.
2. If you forgot to change your preference and want to change it and still go to the safe-zone.
3. The extremely rare instance that atmos has been messed with outside of the shuttle and you want to quickly get back to the shuttle.

It doesn't appear until the round ends, so it doesn't create clutter. You don't need to use the verb in 99% of cases. It's just as a backup if anything.